File: 2차 과제/3-1/3-1.c
#include <stdio.h>

int main() {
    int sale[4]={157, 209, 251, 312};
    for(int i=0;i<4;i++)
        printf("address:%u sale[%d]= %d\n", &(sale[i]), i, sale[i]);
    
    return 0;
}

File: DataStructure/2차 과제/3-1/3-1.py
def main() :
    sale = [157, 209, 251, 312]
    for dex1 in range(4):
        print(f"address : {id(sale[dex1])} sale[{dex1}]= {sale[dex1]}")

if __name__ == "__main__":
    main()